Source/GUI/mainwindow_More.cpp

```diff
@@ -88,6 +88,20 @@ void MainWindow::openCapture()
 }
 #endif // BLACKMAGICDECKLINK_YES
 //---------------------------------------------------------------------------
+bool MainWindow::canCloseFile(size_t index)
+{
+    if(Files[index]->commentsUpdated())
+    {
+        auto result = QMessageBox::warning(this, "Comments has been updated!",
+                                           "Your comments has been modified but were not saveed!"
+                                           "If you close this file all the changed will be lost. Do you really want to close this file?",
+                                           QMessageBox::Yes|QMessageBox::No);
```

Comment:
In my build, the line returns don't render in the dialogue box. Also the "comments has [sic] been updated?" line doesn't seem to appear anywhere.

For dialogue, I suggest "This report contains unsaved comments. If you close before saving, your changes will be lost."

And change buttons to: Cancel | Close. ping to @ablwr